doc/modules/tree.rst

```diff
@@ -363,6 +363,11 @@ Tips on practical use
     classification with few classes, ``min_samples_leaf=1`` is often the best
     choice.
 
+  * Note that ``min_samples_split`` considers samples directly and independent of
```

Comment:
Just to make it part of the previous bullet point in a new paragraph, instead of a new bullet point

```suggestion
    Note that ``min_samples_split`` considers samples directly and independent of
```